<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScrWidgForm</class>
 <widget class="QMainWindow" name="ScrWidgForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="windowTitle">
    <string>Form</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="margin">
     <number>1</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="fieldTabs"/>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QTabWidget" name="tabPanels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>500</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="elideMode">
           <enum>Qt::ElideNone</enum>
          </property>
          <widget class="QWidget" name="tab_2">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Описание</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPlainTextEdit" name="descEdit"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="notesTab">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Ноты</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <widget class="QSplitter" name="splitter_2">
              <property name="frameShadow">
               <enum>QFrame::Plain</enum>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="childrenCollapsible">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="gridLayoutWidget_2">
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="1">
                 <widget class="QToolButton" name="btnAddNote">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="tracks.qrc">
                    <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QToolButton" name="btnDeleteNote">
                  <property name="text">
                   <string>...</string>
                  </property>
                  <property name="icon">
                   <iconset resource="tracks.qrc">
                    <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="0" rowspan="3">
                 <widget class="QTableWidget" name="notesTable">
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="verticalHeaderDefaultSectionSize">
                   <number>19</number>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Дата</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Автор</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Заголовок</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="gridLayoutWidget_3">
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="0" column="1">
                 <widget class="QToolButton" name="btnCommitNote">
                  <property name="text">
                   <string>Ok</string>
                  </property>
                  <property name="icon">
                   <iconset resource="tracks.qrc">
                    <normaloff>:/images/ok_1.png</normaloff>:/images/ok_1.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QToolButton" name="btnRevertNote">
                  <property name="text">
                   <string>Revert</string>
                  </property>
                  <property name="icon">
                   <iconset resource="tracks.qrc">
                    <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="0" rowspan="3">
                 <widget class="QPlainTextEdit" name="noteEdit">
                  <property name="tabChangesFocus">
                   <bool>true</bool>
                  </property>
                  <property name="plainText">
                   <string notr="true"/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="filesTab">
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Файлы</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout">
            <property name="margin">
             <number>3</number>
            </property>
            <property name="spacing">
             <number>3</number>
            </property>
            <item row="0" column="0" rowspan="5">
             <widget class="QTableWidget" name="filesTable">
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Дата</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Автор</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Файл</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="btnAddFile">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="tracks.qrc">
                <normaloff>:/images/edit_add.png</normaloff>:/images/edit_add.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QToolButton" name="btnDeleteFile">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="tracks.qrc">
                <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QToolButton" name="btnOpenFile">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>301</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="1">
             <widget class="QToolButton" name="btnSaveFile">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionEdit_Record"/>
   <addaction name="actionSave_Record"/>
   <addaction name="actionCancel_Changes"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionEdit_Record">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionSave_Record">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/ok.png</normaloff>:/images/ok.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionCancel_Changes">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </action>
  <action name="actionAdd_Note">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/edit_add.png</normaloff>:/images/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить ноту</string>
   </property>
  </action>
  <action name="actionDelete_Note">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
   </property>
   <property name="text">
    <string>Удалить ноту</string>
   </property>
   <property name="toolTip">
    <string>Delete Note</string>
   </property>
  </action>
  <action name="actionCommit_Note_Changes">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/ok_1.png</normaloff>:/images/ok_1.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить изменения в ноте</string>
   </property>
  </action>
  <action name="actionRevert_Note_Changes">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Отменить изменения в ноте</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="tracks.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="tracks.qrc"/>
 </resources>
 <connections/>
</ui>
